sugarlabs/gogo
# Port to GTK3, Python3, glade (need extensive testing)

Test Plan:
* connect a Gogo board (idk if RPi) is supported on OLPC. if it detects one, the activity will continue to the next.
* start the activity and follow the on screen options
* check if all the options are working and the errors are
